tests/drivers/i2s/i2s_api/Kconfig

```diff
@@ -12,3 +12,25 @@ config I2S_TEST_SEPARATE_DEVICES
 	bool "Use two separate I2S ports for loopback"
 	help
 	  Use separate I2S ports for transmit and receive.
+
+config I2S_TEST_USE_I2S_DIR_BOTH
+	bool "Use I2S_DIR_BOTH value to perform RX/TX transfers"
+	help
+	  Use the I2S_DIR_BOTH enumeration value to trigger commands in test
+	  cases involving both reception and transmission. Use of this option
+	  is essential for devices that cannot independently start and stop
+	  the RX and TX streams.
+
+config I2S_TEST_USE_GPIO_LOOPBACK
+	bool "Use GPIO loopback"
+	help
+	  Use wiring between the data-out and data-in pins for looping back
+	  data. This option is intended to be used for devices that do not
+	  provide the internal loopback functionality.
+
```

Comment:
This seems to be same as `config I2S_TEST_SEPARATE_DEVICES`

Reply:
Hello how can i test the following Testcase. I also would like to test I2S peripheral for my nRF52840 dk board.